coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "line": 100.0
```

Comment:
Great work getting simplecov set up. Just a heads up, though, that we don't expect you to set it up on assessments unless you want to! It's up to you!